examples/environments/azure-env.yml

```diff
@@ -0,0 +1,16 @@
+name: azure-live-example
+org_name: runx
+providers:
+  azurerm:
+    location: centralus
+    tenant_id: e1cebc17-5d81-4d16-b435-702e291d395b
+    subscription_id: 0fca99ae-5717-4a86-a891-6d64d2a04788
+modules:
+  - type: base
+  - type: dns
+    domain: azure-example.runx.dev
+    delegated: false
+  - type: k8s-cluster
+    admin_group_object_ids: ["35959136-eccf-44ec-beed-559a8b33c18e"]
```

Comment:
Is this required?

Reply:
no, it isn't, but it's very nice to have because it's a quick way to grant folks in an admin group k8s admin access